tests/installation/welcome.pm

```diff
@@ -33,7 +33,7 @@ sub run {
     # Add tag for untrusted-ca-cert with SMT
     push @welcome_tags, 'untrusted-ca-cert' if get_var('SMT_URL');
     # Add tag for sle15 upgrade mode, where product list should NOT be shown
-    push @welcome_tags, 'inst-welcome-no-product-list' if sle_version_at_least('15') and get_var('UPGRADE');
+    push @welcome_tags, 'inst-welcome-no-product-list' if is_sle('15+') and get_var('UPGRADE');
```

Comment:
this one is interesting. Previously on openSUSE it matched so the tag would be added for check but nobody cared as there is no product view and also there are no openSUSE needles for this tag ;) -> https://openqa.opensuse.org/tests/802961#step/welcome/1 So everything fine as is as well